rectools/model_selection/cross_validate.py

```diff
@@ -48,6 +48,7 @@ def cross_validate(  # pylint: disable=too-many-locals
     models: tp.Dict[str, ModelBase],
     k: int,
     filter_viewed: bool,
+    users: tp.Optional[ExternalIds] = None,
```

Comment:
Hi! Thank you for contribution!

Could you please add the following to your PR:
1. Make `users` parameter as the last last one
2. Add profound testing in `tests/model_selection/test_cross_validate.py`. Please ensure that math is correctly checked there. We believe that tests are really important
3. Add your change in CHANGELOG.md

Reply:
Thanks for the feedback, I'm on it.